conf/messages

```diff
@@ -530,6 +530,30 @@ label.next_page_forgotten_userid=On the next page, go to ‘Problems signing in
 
 label.home_page.utr=Your Self Assessment Unique Taxpayer Reference (UTR) is
 
+#*************************************************************
+# Form error messages radio
+#*************************************************************
+
+error.tax_credits_select = Select yes if you get tax credits
+head.error.tax_credits_select = Select yes if you get tax credits
```

Comment:
Same as above, this `head` pattern seems a bit off

Reply:
that's how it's been previously, I can have a look if I can change it but I'm not sure I can, the error component at the top of the page is from an external library and it expects the message to be head. + error message key

Reply:
All the views are using the same component for the error summary on top of the page which adds the head. prefix to the error message, if I change it to not use that it will affect other messages and screens that were not being changed in this audit